src/Http/Controllers/WebhookController.php

```diff
@@ -48,7 +48,7 @@ public function __invoke(Request $request)
             return new Response();
         }
 
-        $method = 'handle'.Str::studly($payload['alert_name']);
+        $method = 'handle'.Str::studly($payload['event_type']);
```

Comment:
**Note:**

These events include a period "." which has to be removed or else the name ends up like "handleAddress.Created" instead of "handleAddressCreated"

```
 $method = 'handle'.Str::studly(Str::replace('.', ' ', $payload['event_type']));
```

**Full list of events and methods:**

```
address.created -> handleAddressCreated
address.updated -> handleAddressUpdated
adjustment.created -> handleAdjustmentCreated
adjustment.updated -> handleAdjustmentUpdated
business.created -> handleBusinessCreated
business.updated -> handleBusinessUpdated
customer.created -> handleCustomerCreated
customer.updated -> handleCustomerUpdated
discount.created -> handleDiscountCreated
discount.updated -> handleDiscountUpdated
payout.created -> handlePayoutCreated
payout.paid -> handlePayoutPaid
price.created -> handlePriceCreated
price.updated -> handlePriceUpdated
product.created -> handleProductCreated
product.updated -> handleProductUpdated
subscription.activated -> handleSubscriptionActivated
subscription.canceled -> handleSubscriptionCanceled
subscription.created -> handleSubscriptionCreated
subscription.imported -> handleSubscriptionImported
subscription.past_due -> handleSubscriptionPastDue
subscription.paused -> handleSubscriptionPaused
subscription.resumed -> handleSubscriptionResumed
subscription.trialing -> handleSubscriptionTrialing
subscription.updated -> handleSubscriptionUpdated
transaction.billed -> handleTransactionBilled
transaction.canceled -> handleTransactionCanceled
transaction.completed -> handleTransactionCompleted
transaction.created -> handleTransactionCreated
transaction.paid -> handleTransactionPaid
transaction.past_due -> handleTransactionPastDue
transaction.payment_failed -> handleTransactionPaymentFailed
transaction.ready -> handleTransactionReady
transaction.updated -> handleTransactionUpdated
```
```